orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Execution.groovy

```diff
@@ -88,14 +88,14 @@ abstract class Execution<T> implements Serializable {
   }
 
   ExecutionStatus getStatus() {
-    if (canceled) {
-      return CANCELED
-    }
-
     if (version > 1) {
       return executionStatus
     }
 
+    if (canceled) {
```

Comment:
Hmmm ya. 

Makes sense given that we can now end up with pipelines that are technically "canceled = true" but TERMINAL.

You have another PR pending with the schema changes, right? Just pending a migration.